Categories/Category/Construction/Properties/EilenbergMoore.agda

```diff
@@ -0,0 +1,103 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.Properties.EilenbergMoore where
+
+open import Level
+import Relation.Binary.PropositionalEquality as ≡
+
+open import Categories.Adjoint
+open import Categories.Adjoint.Properties
+open import Categories.Category
+open import Categories.Functor using (Functor; _∘F_)
+open import Categories.Functor.Equivalence
+open import Categories.Monad
+
+open import Categories.NaturalTransformation renaming (id to idN)
+
+open import Categories.Adjoint.Construction.EilenbergMoore
+open import Categories.Category.Construction.EilenbergMoore
+
+private
+  variable
+    o ℓ e : Level
+    𝒞 𝒟 : Category o ℓ e
+
+open NaturalTransformation
+
+module _ {F : Functor 𝒞 𝒟} {G : Functor 𝒟 𝒞} (adjoint : Adjoint F G) where
+  T : Monad 𝒞
```

Comment:
all these definitions and modules are now become public. probably they should be under a `private` block